core/src/main/scala/chisel3/Module.scala

```diff
@@ -738,12 +739,56 @@ package experimental {
           case (_, Some(parent)) => parent.toRelativeTarget(root).instOf(this.instanceName, name)
           // If root was defined, and there is no parent, the root was not an ancestor.
           case (Some(definedRoot), None) =>
-            throwException(s"Requested .toRelativeTarget relative to ${definedRoot.name}, but it is not an ancestor")
+            throwException(
+              s"Requested .toRelativeTarget relative to ${definedRoot.name}, but it is not an ancestor of $this"
+            )
           // If root was not defined, and there is no parent, return this.
           case (None, None) => getTarget
         }
     }
 
+    /** Returns a FIRRTL ModuleTarget that references this object, relative to an optional root.
+      *
+      * If `root` is defined, the target is a hierarchical path starting from `root`.
+      *
+      * If `root` is not defined, the target is a hierarchical path equivalent to `toAbsoluteTarget`.
+      *
+      * @note If `root` is defined, and has not finished elaboration, this must be called within `atModuleBodyEnd`.
+      * @note The BaseModule must be a descendant of `root`, if it is defined.
+      * @note This doesn't have special handling for Views.
```

Comment:
What does this comment mean? Since this is only involving Modules, views shouldn't be relevant (but I might be missing something).

Reply:
Oh I see the logic about `ViewParent`, but I think that should never come up since that's only relevant as the parent of views (which are `Data`).

Reply:
I see how that this is [obviously] also implemented for Data, just the same Scaladoc, makes sense!